### DISCUSSION TITLE
ComfyUI installer keeps pausing

### DISCUSSION THREAD

**Discussion by Metaldandy** [Comfy-Org/ComfyUI#127] [Q&A]
I downloaded the standalone and got a couple models and went to run the run_nvidia_gpu.bat and nothing happens for about 30 seconds before it just says

D:\Programs\ComfyUI>.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build

D:\Programs\ComfyUI>pause
Press any key to continue . . .

Is there anything else I need before running this?

**Comment by comfyanonymous** [comment]
There's no error or anything? That's strange...

Does the run_cpu.bat work?

You can also try the older pytorch 1.13 version on this page: https://github.com/comfyanonymous/ComfyUI/releases

**Comment by loredane1** [comment]
I had the same problem and it was the configuration of the "extra_model_paths.yaml" file that was inconsistent. Try deactivating it and opening Confyui to see if the error comes from it. If it is, you'll have to revise the format of the path you've entered
